xml/System.Text.RegularExpressions/GroupCollection.xml

```diff
@@ -1448,10 +1378,10 @@
         <ReturnType>System.Collections.Generic.IEnumerable&lt;System.Text.RegularExpressions.Group&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets a group enumeration with all the groups in the group collection.</summary>
+        <value>A group enumeration.</value>
         <remarks>To be added.</remarks>
```

Comment:
Add to the Remarks that this is an EII -- you can copy and adapt the earlier text.